src/h3lib/lib/faceijk.c

```diff
@@ -593,7 +594,8 @@ void _faceIjkPentToGeoBoundary(const FaceIJK* h, int res, GeoBoundary* g) {
                     edge0 = &v1;
                     edge1 = &v2;
                     break;
-                case KI:
+                default:
```

Comment:
Any thoughts on having:
```
case KI:
default:
    assert(... = KI); // Ensure we did not get here via the default label.
```

The idea here is that it uses the normal, expected idiom to go to KI, maintains the assertion, and doesn't require adding uncoverable lines.

Reply:
OK. I mentioned above that the Google developers working on opencensus disagree with me entirely that there needs to be a default. There is a clang flag for it.